src/GameEngine/Entity/Component.cpp

```diff
@@ -223,4 +307,4 @@ void ge::GameObject::setTag(std::string const & t_tag)
 std::string ge::GameObject::getTag() const
 {
```

Comment:
why not return a const reef here ? it is better to let the user decide if he wants a copy or net